packages/faceted-search/src/dictionary/operator.dictionary.js

```diff
@@ -14,14 +14,14 @@ const standardOperators = t => ({
 		label: t('OPERATOR_NOT_EQUALS_LABEL', {
 			defaultValue: 'Not equals',
 		}),
-		name: 'notEqual',
+		name: 'notEquals',
 		iconName: 'not-equal',
 	},
 	[operatorNames.equal]: {
 		label: t('OPERATOR_EQUALS_LABEL', {
 			defaultValue: 'Equals',
 		}),
-		name: 'equal',
+		name: 'equals',
```

Comment:
Just to understand, why you don't change also `greaterThanOrEqual` to `greaterThanOrEquals` and `lessThanOrEqual` to `lessThanOrEquals` here?